asv_bench/benchmarks/replace.py

```diff
@@ -36,6 +36,23 @@ def time_replace_series(self, inplace):
         self.s.replace(self.to_rep, inplace=inplace)
 
 
+class ReplaceList:
+    # GH#28099
+
+    params = [(True, False)]
+    param_names = ["inplace"]
+
+    def setup_cache(self, inplace):
```

Comment:
I think having `inplace` here is a problem as `setup_cache` is not parametrized. Could remove that argument though I think better to just change to `setup` as this construction doesn't take that long